resources/event-bus/charts/subscription-controller-knative/templates/_helpers.tpl

```diff
@@ -6,6 +6,22 @@
 {{- printf "%s-%s" .Release.Name "subscription-controller-knative" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{- define "subscription-controller-knative-metrics.name" -}}
+{{- printf "%s-%s" .Release.Name "subscription-controller-knative-metrics" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "subscription-controller-knative-metrics-service.name" -}}
+{{- printf "%s-%s" .Release.Name "subscription-controller-knative-metrics-service" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "subscription-controller-knative-metrics-service-monitor.name" -}}
+{{- printf "%s-%s" .Release.Name "subscription-controller-knative-metrics-service-monitor" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "subscription-controller-knative-metrics-destination-rule.name" -}}
+{{- printf "%s-%s" .Release.Name "subscription-controller-knative-metrics-destination-rule" | trunc 63 | trimSuffix "-" -}}
```

Comment:
I think that the **.Release.Name "subscription-controller-knative-metrics-destination-rule"** will graw above 63 characters, please check that after truncation there will be no problems.

Reply:
It will, but with no harm